### ISSUE TITLE
Plugin for Intellij

### ISSUE THREAD

**Issue by ppamorim** [jitpack/jitpack.io#48]
Will be good to have a plugin to warning when the library have a update or structure change. Look like a log.

**Comment by jitpack-io** [comment]
Interesting! Do you mean something like VersionEye?

**Comment by ppamorim** [comment]
Yes, but it will do a research every hour or when the user click on the button.

**Comment by jitpack-io** [comment]
That's an awesome idea, no doubt. Additionally the notifications could also be provided by a Gradle plugin.

**Comment by lokkju** [comment]
@ppamorim do you just mean a "hey, this library you are using has been updated" notification, combined with some sort of binary compatibility checking?  I'm not sure if I'd find that highly useful myself, but if you wanted to do something like that, https://github.com/jeluard/semantic-versioning would be a good starting point for the compatibility (and semver) checking.

**Comment by shkschneider** [comment]
Update notification would be HUGE! +1
